spg/Animal/bin/barnyard.pl

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+use Animal;
+use Cow;
+use Horse;
+use Sheep;
+use Mouse;
+
+print "Build your own farm!\n";
+
+my @barnyard
+
+while ( my $input = <STDIN> ) {
+  chomp( $input ); 
+  push @barnyard, $input;
+}
```

Comment:
@Utopia1138 That would just set barnyard to an array of 1's, since chomp returns the number of removed characters. It should be
```perl
chomp( my @barnyard = <STDIN> );
```